MerchantSDK/GiniMerchantSDK/Sources/GiniMerchantSDK/Core/PaymentInfo.swift

```diff
@@ -19,4 +20,13 @@ public struct PaymentInfo {
     public var paymentUniversalLink: String
     public var paymentProviderId: String
 
+    public init(recipient: String, iban: String, bic: String, amount: String, purpose: String, paymentUniversalLink: String, paymentProviderId: String) {
```

Comment:
Do we need this init - structs has default constructor

Reply:
As long as you do not implement init() explicitly, it is marked as internal by default, and we need it public for being able to used it from ExampleApp. 
Thanks 👍